pkg/util/strings.go

```diff
@@ -0,0 +1,27 @@
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package util
+
+import "strings"
+
+// InStringSlice checks whether a string is inside a string slice.
+// Comparison is case insensitive.
+func InStringSlice(ss []string, str string) bool {
```

Comment:
nit: this should be in existing strutil.go . Also please clarify the origin of the code in the comment.

Reply:
@AkihiroSuda  what do you mean by `existing strutil.go` ? in nerdctl project or in other project ?

Reply:
https://github.com/AkihiroSuda/nerdctl/blob/master/strutil.go